method/link/create.js

```diff
@@ -19,10 +20,40 @@ module.exports = function CreateFeedGroupLink (server) {
 
     if (!name) delete content.name
 
-    if (!isValid(content)) return cb(isValid.errors)
+    if (!isValidFeedGroupLink(content)) return cb(isValidFeedGroupLink.errors)
 
     server.publish(content, cb)
     // NOTE we *could* check the group is for a group we know about
     // but if it's not, encryption based on recps will fail
   }
 }
+
+function CreateGroupSubgroupLink (server) {
+  return function createGroupSubgroupLink ({ group, subgroup, kaitiaki }, cb) {
+    if (!isGroupId(group)) return cb(new Error('link.create expects a valid groupId for the group'))
+    if (!isGroupId(subgroup)) return cb(new Error('link.create expects a valid groupId for the subgroup'))
+
+    const content = {
+      type: 'link/group-group',
+      parent: group,
+      child: subgroup,
+      kaitiaki: { set: kaitiaki },
+      subgroup: { set: true },
```

Comment:
added this flag here so we can determine whether the link is for a group or subgroup

Reply:
please write this with ssb-crut (or ssb-crut-authors?)
also replace the other one. These modules put in a lot of checks for free which means we can worry about other things

Reply:
thinking about type more it would be easier for querying if the type was `link/group-group/subgroup` or `link/group-subgroup` because then we can look at just one field to know if it's the sort of link we want.

What is the kaitiaki field?

Reply:
could kaitiaki just be the publicly acknowledged authors (using ssb-crut-authors)

Reply:
kaitiaki in this case is a boolean to flag the kaitiaki-only subgroup, but from comments earlier i have now removed it!

Reply:
I think i'll use `link/group-subgroup` its consistent with our other links